vendor/github.com/openshift/library-go/pkg/operator/encryption/controllers/migrators/inprocess.go

```diff
@@ -133,7 +133,9 @@ func (m *InProcessMigrator) runMigration(gvr schema.GroupVersionResource, writeK
 			if err != nil {
 				klog.Warningf("List of %v failed: %v", gvr, err)
 				if errors.IsResourceExpired(err) {
-					return nil, err // the pager will handle that
+					// start a new list without the continue parameter
+					opts.Continue = ""
+					continue
```

Comment:
isn't the pager doing exactly this? I don't want to interfere with the pager logic.

Reply:
I tested it locally, without this fix all subsequent requests result in an error.

Reply:
eventually, we will enter the degraded state and we will wait 5 minutes and we will re-run that code but only a few times until we get the error one more time.